README.md

```diff
@@ -1,35 +1,37 @@
 # rhoso-luna-hsm Role
 
-In order to use HSMs, the barbican images need to be customized to include the HSM software.
+In order to use Thales Luna Network HSMs as a PKCS#11 backend, the Barbican
+images need to be customized to include the [Luna HSM Client](https://www.thalesdocs.com/gphsm/luna/7/docs/network/Content/Home_Luna.htm) software.
 
 The purpose of this role is to:
-* Generate new images for the barbican-api and barbican-worker containing the HSM software
-* Upload those images to a private repository for use in setting up a CI job.
-* Create any required config to be mounted by the barbican images to interact with the HSM
+* Generate new images for the barbican-api and barbican-worker containing the
+  [Luna Minimal Client](https://www.thalesdocs.com/gphsm/luna/7/docs/network/Content/install/client_install/linux_docker_minimal_extended.htm)
+* Upload those images to a private repository for use in a RHOSO deployment.
+* Create any required config to be mounted by the barbican images for connecting to the HSMs
+  using [NTLS](https://www.thalesdocs.com/gphsm/luna/7/docs/network/Content/admin_partition/connections/connections.htm#NTLS)
 
-For the Lunasa, we expect some preparatory steps to be completed prior to execution in order for the
-role to complete successfully.
-* The lunasa software is uploaded somewhere and will be fetched by the role
-  * The contents of the minimal linux client in a zipped tar file should be made available at luna_minclient_src.
-  * The lunasa binaries that need to be added to the image are made available at luna_binaries_src.
-  * The lunasa HSM cacert file is made available at luna_server_cert_src.  For an HA configuration,
-    this will be a concatenation of all the server certs for the servers in the HA partition.
+We expect some preparatory steps to be completed prior to execution in order for the
+role to complete successfully:
+* The Luna Minimal Client tarball has been downloaded from Thales
+  * The location of the Luna Minimal Client tarball should be made available at luna_minclient_src.
+  * The HSM Appliance Server Certificate (server.pem) file is made available at luna_server_cert_src.
+    When using more than one HSM appliance, this file will be a concatenation of all the server certificates.
   * The client certificate and key made available at luna_client_cert_src.  The files are expected
-    to be of the form "(client_ip)".pem and "(client_ip)"Key.pem
+    to be of the form $(CLIENT_NAME).pem and $(CLIENT_NAME)Key.pem
```

Comment:
We've made this change here, but have not changed the parameter - which is still called client_ip.
This parameter probably needs some explanation as well.

Reply:
Yeah, sorry I started to change this when I updated the doc but haven't gotten to the actual variable yet.  I mainly want to rename this because `client_ip` implies that it has to be an IP, but this could be either a hostname or an IP.  I was thinking `client_host` at first since host can be either a hostname or IP.